extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientRuntimeConfig.java

```diff
@@ -66,12 +87,30 @@ public class InfinispanClientRuntimeConfig {
      * Sets username used by authentication.
      */
     @ConfigItem
+    Optional<String> username;
+
+    /**
+     * Sets username used by authentication.
+     *
+     * @deprecated {@link #username} should be used to configure the credentials username.
+     */
+    @ConfigItem
+    @Deprecated
```

Comment:
Same as above

Reply:
I added the javadoc deprecated these ones